config/initializers/secure_headers.rb

```diff
@@ -25,9 +25,7 @@
     https://www.googletagmanager.com/gtm.js
   ]
 
-  # curl https://www.google.com/supported_domains > config/csp/google_supported_domains.csv
```

Comment:
This is fine but I have no idea why of all of Google's domains `.com.sa` is special 🤷🏽‍♂️

Reply:
That was there before the supported domains list was pulled in so I figured it was one that was commonly popping up 🤔 

Just removed it!

Reply:
Yeah - I remember it being there before, I just never understood why it was. I think reverting back to what you've left in is sensible and we can just keep an eye on it and add some back if needs be 👍🏽